demo/electron-tabs.html

```diff
@@ -17,7 +17,12 @@
     //const TabGroup = require('electron-tabs') normally but for demo :
     const TabGroup = require("../index");
 
-    let tabGroup = new TabGroup();
+    let tabGroup = new TabGroup({
+        newTab: {
+            title: 'New Tab',
+            icon: 'fa fa-search'
```

Comment:
Why `icon: 'fa fa-search'`? 
It looks useless since demo does not include font-awesome.

Reply:
You're right. I have remove it.